public/admin.js

```diff
@@ -325,12 +341,11 @@ export default function() {
     });
   }
 
-  switch (window.location.pathname) {
-    case '/admin':
-      return mainAdminPanel();
-      break;
-    case '/admin/locale':
-      return localeAdminPanel();
-      break;
-  }
+  var path = window.location.pathname;
```

Comment:
Was this an automatic conversion or did you change it manually?

Reply:
I did change it. The path has to end with `/something` instead of being `/something`.